locale/be_BY.ts

```diff
@@ -4781,109 +4723,109 @@ of the appropriate groups to use them.</source>
         <source>Use of Forvo currently requires an API key. Leave this field
 blank to use the default key, which may become unavailable
 in the future, or register on the site to get your own key.</source>
-        <translation>Карыстаньне сайтам Forvo патрабуе ключ API. Пакінь поле пустым, каб карыстацца стандартным ключом, які пазьней можа стаць недаступным, ці зарэґіструйся на сайце і атрымай уласны ключ.</translation>
+        <translation>Для карыстання сайтам Forvo патрэбны ключ API. Пакінце поле пустым, каб выкарыстоўваць прадвызначаны ключ, які пазней можа стаць недаступным, або зарэгіструйцеся на сайце і атрымайце ўласны ключ.</translation>
```

Comment:
When source text is split into several lines, the translation should generally be split as well. This usually means a tooltip, which doesn't look good or even doesn't fit the screen width, if too long.

@antikruk, could you please correct this issue here and in other places?